tests/functional/sign_up_coppa_input.js

```diff
@@ -0,0 +1,128 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+define([
+  'intern',
+  'intern!object',
+  'intern/chai!assert',
+  'require',
+  'tests/lib/helpers',
+  'tests/functional/lib/helpers'
+], function (intern, registerSuite, assert, require, TestHelpers, FunctionalHelpers) {
+  /* global $ */
+
+  var PAGE_URL = intern.config.fxaContentRoot + 'signup?forceCoppa=input';
+  var CUTOFF_AGE = 13;
+  var email;
+  var PASSWORD = '12345678';
+
+  registerSuite({
+    name: 'sign_up input_coppa',
+
+    beforeEach: function () {
+      email = TestHelpers.createEmail();
+      return FunctionalHelpers.clearBrowserState(this);
+    },
+
+    afterEach: function () {
+      return FunctionalHelpers.clearBrowserState(this);
+    },
+
+    'sign up with COPPA input properly submits if age is valid': function () {
+      return this.remote
+        .setFindTimeout(intern.config.pageLoadTimeout)
+        .get(require.toUrl(PAGE_URL))
+
+        .findByCssSelector('.email')
+        .type(email)
+        .end()
+
+        .findByCssSelector('.password')
+        .type(PASSWORD)
+        .end()
+
+        .findByCssSelector('#submit-btn')
+        .click()
+        .end()
+
+        // error tooltip works
+        .then(FunctionalHelpers.visibleByQSA('#coppa .tooltip'))
+        .findByCssSelector('#coppa .tooltip')
+        .getVisibleText()
```

Comment:
Can you indent the contents between `find` and `end` to match other functional test modules?